### ISSUE TITLE
Erroneous rule for FROM statement

### ISSUE THREAD

**Issue by onlinejudge95** [replicatedhq/dockerfilelint#144]
For a multistage build, the `dockerfilelint` reports the following error
```
First Command Must Be FROM (Possible Bug)

The first instruction in a Dockerfile must specify the base image using a FROM command. Additionally, FROM cannot appear later in a Dockerfile.
```

Here is the `Dockerfile` I used

https://github.com/onlinejudge95/rinnegan/blob/master/services/server/Dockerfile

**Comment by SimonHeimberg** [comment]
On what version did you find this?
  * https://www.fromlatest.io/#/ , the original online one, which is outdated (I can not see the version)
  * http://www.dockerfilelint.com/#/ , which uses a newer version of this tool (1.4.0). It supports multistage dockerfiles
  * local version xxx

**Comment by onlinejudge95** [comment]
In https://www.fromlatest.io/

Thanks for clarifying it is outdated, can you update the current repo's ABout section with the same

**Comment by SimonHeimberg** [comment]
https://www.fromlatest.io/ seems to have updated the version. At least it does not report this failure any more. So I currently have no idea which online checker is newer.
